LeaderboardBackend.Test/Features/Users/TestRecoveryTests.cs

```diff
@@ -0,0 +1,160 @@
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using LeaderboardBackend.Models.Entities;
+using LeaderboardBackend.Test.Fixtures;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.DependencyInjection;
+using NodaTime;
+using NodaTime.Testing;
+using NUnit.Framework;
+
+namespace LeaderboardBackend.Test.Features.Users;
+
+public class TestRecoveryTests : IntegrationTestsBase
+{
+    private IServiceScope _scope = null!;
+    private readonly HttpClient _client;
+    private readonly FakeClock _clock = new(Instant.FromUnixTimeSeconds(1));
+
+    public TestRecoveryTests()
+    {
+        _client = _factory.WithWebHostBuilder(builder =>
```

Comment:
Avoid using the constructor with NUnit and use a [OneTimeSetUp] method instead.
As opposed to XUnit, NUnit doesn't guarantee that it will run the constructor only once.